--- lxc-1.1.5/src/lxc/conf.c.orig	2015-11-09 16:25:59.000000000 +0000
+++ lxc-1.1.5/src/lxc/conf.c	2018-08-30 08:15:50.237971492 +0000
@@ -33,6 +33,7 @@
 #include <sys/wait.h>
 #include <sys/syscall.h>
 #include <sys/types.h>
+#include <sys/sysmacros.h>	// for makedev() because since glibc 2.25 the macros “major”, “minor”, and “makedev” are not included from <sys/tyupes.h>
 #include <pwd.h>
 #include <grp.h>
 #include <time.h>
--- lxc-1.1.5/src/lxc/lxccontainer.c.orig	2015-11-09 16:25:59.000000000 +0000
+++ lxc-1.1.5/src/lxc/lxccontainer.c	2018-08-30 08:21:52.985645702 +0000
@@ -24,6 +24,7 @@
 #include <pthread.h>
 #include <unistd.h>
 #include <sys/types.h>
+#include <sys/sysmacros.h>	// for makedev() because since glibc 2.25 the macros “major”, “minor”, and “makedev” are not included from <sys/tyupes.h>
 #include <sys/wait.h>
 #include <sys/mount.h>
 #include <errno.h>
--- lxc-1.1.5/src/lxc/bdev.c.orig	2015-11-09 16:25:59.000000000 +0000
+++ lxc-1.1.5/src/lxc/bdev.c	2018-08-30 08:23:08.905158838 +0000
@@ -32,6 +32,7 @@
 #include <stdint.h>
 #include <inttypes.h>
 #include <sys/types.h>
+#include <sys/sysmacros.h>	// for makedev() because since glibc 2.25 the macros “major”, “minor”, and “makedev” are not included from <sys/tyupes.h>
 #include <grp.h>
 #include <unistd.h>
 #include <errno.h>
