--- ganeti-2.15.2/lib/client/gnt_instance.py.orig	2015-12-15 14:55:37.000000000 +0000
+++ ganeti-2.15.2/lib/client/gnt_instance.py	2018-09-14 06:09:32.036735200 +0000
@@ -69,7 +69,7 @@
 
 #: default list of options for L{ListInstances}
 _LIST_DEF_FIELDS = [
-  "name", "hypervisor", "os", "pnode", "status", "oper_ram",
+  "name", "hypervisor", "os", "pnode", "status", "oper_ram", "disk.size/0",
   ]
 
 _MISSING = object()
