--- ./lib/impexpd/__init__.py.orig	2016-01-03 08:45:52.629334264 +0000
+++ ./lib/impexpd/__init__.py	2016-01-03 08:46:01.169334258 +0000
@@ -88,7 +88,7 @@
 
 # Common options for socat
 SOCAT_TCP_OPTS = ["keepalive", "keepidle=60", "keepintvl=10", "keepcnt=5"]
-SOCAT_OPENSSL_OPTS = ["verify=1", "method=TLS1",
+SOCAT_OPENSSL_OPTS = ["verify=0", "method=TLS1",
                       "cipher=%s" % constants.OPENSSL_CIPHERS]
 
 if constants.SOCAT_USE_COMPRESS:
