--- ./lib/hypervisor/hv_lxc.py.orig	2016-02-25 10:15:57.580669514 +0000
+++ ./lib/hypervisor/hv_lxc.py	2016-02-25 10:19:13.485302983 +0000
@@ -579,7 +579,7 @@
     out.append("lxc.mount.entry = sysfs sys sysfs defaults 0 0")
     out.append("lxc.mount.entry = none dev/shm tmpfs rw,nosuid,nodev,create=dir") # See /usr/share/lxc/config/gentoo.common.conf
 
-    # CPUs
+    # pinned CPUs
     if instance.hvparams[constants.HV_CPU_MASK]:
       cpu_list = utils.ParseCpuMask(instance.hvparams[constants.HV_CPU_MASK])
       cpus_in_mask = len(cpu_list)
@@ -591,6 +591,15 @@
                                       cpus_in_mask))
       out.append("lxc.cgroup.cpuset.cpus = %s" %
                  instance.hvparams[constants.HV_CPU_MASK])
+    # not pinned, but limited CPUs
+    else:
+      # https://www.kernel.org/doc/Documentation/scheduler/sched-bwc.txt
+      # By using a small period here we are ensuring a consistent latency response
+      period = 50000
+      out.append("# capped to %s CPUs" % instance.beparams["vcpus"])
+      out.append("lxc.cgroup.cpu.cfs_period_us = %s" % period)
+      quota = period * instance.beparams["vcpus"]
+      out.append("lxc.cgroup.cpu.cfs_quota_us = %s" % quota)
 
     # Memory
     out.append("lxc.cgroup.memory.limit_in_bytes = %dM" %
