--- ./lib/hypervisor/hv_lxc.py.orig	2016-09-23 08:59:53.666652067 +0000
+++ ./lib/hypervisor/hv_lxc.py	2016-09-23 09:53:04.560508130 +0000
@@ -577,6 +577,7 @@
     # Necessary file systems
     out.append("lxc.mount.auto = proc:mixed sys:mixed cgroup:mixed")
     out.append("lxc.mount.entry = none dev/shm tmpfs rw,nosuid,nodev,create=dir") # See /usr/share/lxc/config/gentoo.common.conf
+    out.append("lxc.include = /usr/local/share/lxc/config/common.conf.d/x-lxcfs-proc-only.conf") # This is almost the same as /usr/share/lxc/config/common.conf.d/00-lxcfs.conf https://github.com/lxc/lxcfs
 
     # CPUs
     if instance.hvparams[constants.HV_CPU_MASK]:
