diff --git a/lib/hypervisor/hv_lxc.py b/lib/hypervisor/hv_lxc.py
index 514896f..38d180a 100644
--- a/lib/hypervisor/hv_lxc.py
+++ b/lib/hypervisor/hv_lxc.py
@@ -577,6 +577,7 @@ class LXCHypervisor(hv_base.BaseHypervisor):
     # Necessary file systems
     out.append("lxc.mount.entry = proc proc proc nodev,noexec,nosuid 0 0")
     out.append("lxc.mount.entry = sysfs sys sysfs defaults 0 0")
+    out.append("lxc.mount.entry = none dev/shm tmpfs rw,nosuid,nodev,create=dir") # See /usr/share/lxc/config/gentoo.common.conf
 
     # CPUs
     if instance.hvparams[constants.HV_CPU_MASK]:
