diff --git a/MANIFEST.in b/MANIFEST.in
index bd94f4e..dc3e52e 100644
--- a/MANIFEST.in
+++ b/MANIFEST.in
@@ -1,11 +1,2 @@
 include LICENSE
 include MANIFEST.in
-include check-dependencies.py
-include examples/*
-include conf/*.example
-include webapp/graphite/local_settings.py.example
-recursive-include distro/ *
-recursive-include webapp/graphite/ *.html
-recursive-include webapp/content/ *
-exclude webapp/graphite/local_settings.py
-exclude conf/*.conf
