diff --git a/webapp/graphite/settings.py b/webapp/graphite/settings.py
index 8e402f4..86ef599 100644
--- a/webapp/graphite/settings.py
+++ b/webapp/graphite/settings.py
@@ -106,7 +106,7 @@ USE_REMOTE_USER_AUTHENTICATION = False
 REMOTE_USER_BACKEND = "" # Provide an alternate or subclassed backend
 
 # Django 1.5 requires this so we set a default but warn the user
-SECRET_KEY = 'UNSAFE_DEFAULT'
+SECRET_KEY = os.urandom(32)
 
 # Django 1.5 requires this to be set. Here we default to prior behavior and allow all
 ALLOWED_HOSTS = [ '*' ]
